checks.d/daemontools.py

```diff
@@ -0,0 +1,39 @@
+# project
+from checks import AgentCheck
+import re
+import subprocess
+
+dt_status = re.compile(".*: (down|up) \(pid (\d+)\) (\d+) seconds")
+
+class DaemonToolsCheck(AgentCheck):
+    def check(self, instance):
+        path = self.init_config.get('path', '/etc/service')
```

Comment:
We are considering deprecating init_config at a point. Can you pass that at the instance level instead ?